projects/1.20/assets/xtra-arrows/xtraarrows/lang/zh_cn.json

```
@@ -0,0 +1,305 @@
+
+    "entity.xtraarrows.diamond_life_steal_arrow": "生命偷取钻石箭",
+    "entity.xtraarrows.iron_life_steal_arrow": "生命偷取铁箭",
+    "entity.xtraarrows.flint_life_steal_arrow": "生命偷取箭",
+    "entity.xtraarrows.golden_life_steal_arrow": "生命偷取金箭",
+    "entity.xtraarrows.netherite_life_steal_arrow": "生命偷取下界合金箭",
+
+    "entity.xtraarrows.diamond_gravity_arrow": "失重钻石箭",
+    "entity.xtraarrows.iron_gravity_arrow": "失重铁箭",
+    "entity.xtraarrows.flint_gravity_arrow": "失重箭",
+    "entity.xtraarrows.golden_gravity_arrow": "失重金箭",
+    "entity.xtraarrows.netherite_gravity_arrow": "失重下界合金箭",
+
+    "effect.xtraarrows.magnetized" : "磁化",
+
+    "itemGroup.xtraarrows": "额外箭矢",
+
+    "xtraarrows.text.netherite_damage": "\u00A7o\u00A79+6 伤害",
+    "xtraarrows.text.diamond_damage": "\u00A7o\u00A79+4 伤害",
+    "xtraarrows.text.golden_damage1": "\u00A7o\u00A79在下界 +3 伤害 ",
+    "xtraarrows.text.golden_damage2": "\u00A7o\u00A79不在下界 -1 伤害 ",
+    "xtraarrows.text.iron_damage": "\u00A7o\u00A79+2 伤害",
+    "xtraarrows.text.padded_desc": "\u00A7o\u00A77无伤害, 命中生物或方块后反弹",
+    "xtraarrows.text.howtounleash": "\u00A7o\u00A76Shift + 鼠标右击 释放被栓生物",
+    "xtraarrows.text.tracking_arrow": "\u00A7o\u00A76追踪最后攻击的生物",
+    "xtraarrows.text.life_steal_arrow": "\u00A7o\u00A76偷取\u00A7f\u00A7r 1/%d §o§6目标生命值(最大10颗心)",
+    "xtraarrows.text.cupids_desc": "\u00A7o\u00A76驯服着弹点附近的可驯服生物",
+    "xtraarrows.text.extinguishing_desc": "\u00A7o\u00A76灭火, 将熔岩转化为黑曜石, 对怕水的生物造成伤害",
+    "xtraarrows.text.incendiary_desc": "\u00A7o\u00A76将水转化为圆石, 使附近的方块和生物着火",
+    "xtraarrows.text.gravity_arrow": "\u00A7o\u00A76箭头无视重力 10 秒",
+    "xtraarrows.text.apple_arrow": "\u00A7o\u00A76给命中的玩家或生物吃一个苹果",
+    "xtraarrows.text.notch_apple_arrow": "\u00A7o\u00A76给命中的玩家或生物吃一个附魔金苹果",
+    "xtraarrows.text.golden_apple_arrow": "\u00A7o\u00A76给命中的玩家或生物吃一个金苹果",
+    "xtraarrows.text.breeding_arrow": "\u00A7o\u00A76繁育着弹点附近可繁育的生物",
+    "xtraarrows.text.atlantean_arrow": "\u00A7o\u00A76在水下射出的箭无视阻力",
+    "xtraarrows.text.ender_arrow": "\u00A7o\u00A76将射击者传送至弹着点",
+    "xtraarrows.text.explosive_arrow": "\u00A7o\u00A76在弹着点产生爆炸",
+    "xtraarrows.text.freezing_arrow": "\u00A7o\u00A76使弹着点附近的生物持续冻伤, 在附近的方块上产生雪",
+    "xtraarrows.text.lightning_arrow": "\u00A7o\u00A76在弹着点召唤一道闪电",
+    "xtraarrows.text.magnetic_arrow1": "\u00A7o\u00A76朝弹着点拉动生物 5 秒",
+    "xtraarrows.text.magnetic_arrow2": "\u00A7o\u00A76使命中的生物磁化",
+    "xtraarrows.text.torch_arrow": "\u00A7o\u00A76在弹着点放置火把",
+    "xtraarrows.text.redstone_torch_arrow": "\u00A7o\u00A76在弹着点放置红石火把",
+    "xtraarrows.text.soul_torch_arrow": "\u00A7o\u00A76在弹着点放置灵魂火把",
+    "xtraarrows.text.slime_arrow": "\u00A7o\u00A76弹力十足!",
+    "xtraarrows.text.lantern_arrow": "\u00A7o\u00A76在弹着点放置灯笼",
+    "xtraarrows.text.soul_lantern_arrow": "\u00A7o\u00A76在弹着点放置灵魂灯笼",
+    "xtraarrows.text.vexing_arrow": "\u00A7o\u00A76穿透一切!",
+
+    "xtraarrows.text.general_disabled": "\u00A7c如果启用, 只有原版箭可以射出",
+    "xtraarrows.text.diamond_disabled": "\u00A7c这个服务器禁止了所有 钻石箭.",

```

```suggestion
    "xtraarrows.text.diamond_disabled": "\u00A7c这个服务器禁用了所有钻石箭。",
```
注意删除多余空格，全角句号，全文通查
禁止推荐改为禁用